### PR TITLE
Document AWS Gov cloud STS authentication

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds documentation on how to use AWS STS with AWS Gov cloud accounts.
💻 [Preview](https://deploy-preview-1113--docs-spectrocloud.netlify.app/clusters/public-cloud/aws/add-aws-accounts)
🎫 [PCP-711](https://spectrocloud.atlassian.net/browse/PCP-711)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1113#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws/05-add-aws-accounts.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/05-add-aws-accounts.md:32]:
```suggestion
To add an AWS cloud account using static access credentials follow these steps:
```

File: content/docs/04-clusters/01-public-cloud/01-aws/05-add-aws-accounts.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/05-add-aws-accounts.md:196]:
```suggestion
You can validate the account is available in Palette by reviewing the list of cloud accounts. To review the list of cloud accounts navigate to the left **Main Menu**. Click on **Tenant Settings**. Next, click **Cloud Accounts**. Your newly added AWS cloud account is listed under the AWS sections.
```

**Review by ritawatson — APPROVED** [review]
Two tiny suggestions. LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1113#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1113#pullrequestreview-1287727658" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
